src/components/getFromApi/GetReviewListOfHotel.jsx

```diff
@@ -0,0 +1,12 @@
+import axios from './AxiosConfig'
+
+////id - of object 
+//// loc -location 
+const GetReviewsListOfHotel = (args) =>{
+    return axios.get('https://hotels4.p.rapidapi.com/reviews/list',{
```

Comment:
I can't find AxiosConfig.
You need to create a folder "services" and move this file

Reply:
i'm create folder "services" and move file here